craft_application/application.py

```diff
@@ -300,6 +301,17 @@ def cache_dir(self) -> pathlib.Path:
                 f"Unable to create/access cache directory: {err.strerror}"
             ) from err
 
+    def _bootstrap_services(self) -> None:
```

Comment:
I'd like to suggest `_configure_early_services()`, so that the difference between it and `_configure_services` is more evident

Reply:
Good idea!